api/src/test/resources/org/openmrs/api/include/ObsServiceTest-initial.xml

```diff
@@ -23,15 +23,16 @@
   <obs obs_id="3" person_id="6" concept_id="1" status="FINAL" obs_datetime="2006-02-11 00:00:00.0" location_id="1" value_numeric="3.0" creator="1" date_created="2006-02-12 15:57:35.0" voided="false" uuid="efbba621-51ca-428b-8671-4f4eede10b4b" accession_number="AN1"/>
   <obs obs_id="4" person_id="6" concept_id="1" status="FINAL" order_id="42" obs_datetime="2006-02-12 00:00:00.0" location_id="1" value_numeric="4.0" creator="1" date_created="2006-02-13 15:57:35.0" voided="false" uuid="144cb55d-69ab-4009-8256-f4750e8770ee" accession_number="AN2"/>
   <obs obs_id="5" person_id="7" concept_id="1" status="FINAL" obs_datetime="2006-02-13 01:00:00.0" location_id="1" value_numeric="5.0" creator="1" date_created="2006-02-14 15:57:35.0" voided="false" uuid="6e76c1f1-6886-49bb-9dc0-f3b304936100"/>
-  <obs obs_id="6" person_id="7" concept_id="1" status="FINAL" obs_datetime="2006-02-14 00:00:00.0" location_id="1" value_numeric="6.0" creator="1" date_created="2006-02-15 15:57:35.0" voided="false" uuid="14d3b79b-1f8f-4bb3-93c2-3eac710dc053"/>
-  <obs obs_id="7" person_id="8" concept_id="1" status="FINAL" obs_datetime="2006-02-15 00:00:00.0" location_id="1" value_numeric="7.0" creator="1" date_created="2006-02-16 15:57:35.0" voided="false" uuid="adf212b7-05bd-4114-908a-b04cb058afa4" form_namespace_and_path="valid_form_namespace^path"/>
+  <obs obs_id="6" person_id="7" concept_id="1" status="FINAL" obs_datetime="2006-02-14 00:00:00.0" location_id="1" value_numeric="6.0" creator="1" date_created="2006-02-15 15:57:35.0" voided="false" uuid="14d3b79b-1f8f-4bb3-93c2-3eac710dc053" form_namespace_and_path="valid_form_namespace^path"/>
+  <obs obs_id="7" person_id="8" concept_id="1" status="FINAL" obs_datetime="2006-02-15 00:00:00.0" location_id="1" value_numeric="7.0" creator="1" date_created="2006-02-16 15:57:35.0" voided="false" uuid="adf212b7-05bd-4114-908a-b04cb058afa4" />
   <obs obs_id="8" person_id="8" concept_id="1" status="FINAL" obs_datetime="2006-02-16 00:00:00.0" location_id="1" value_numeric="8.0" creator="1" date_created="2006-02-17 15:57:35.0" voided="false" uuid="0ee1248e-08aa-4a2c-9f38-fb3875f605e0"/>
-  <obs obs_id="9" person_id="9" concept_id="1" status="FINAL" obs_datetime="2004-01-01 00:00:00.0" location_id="1" value_numeric="9.0" obs_group_id="2" creator="1" date_created="2006-02-17 15:57:35.0" voided="false" uuid="0ee1248e-08aa-4a2c-9f38-fb3875f605e1"/>
```

Comment:
What is the reason behind changing the person_id from 9 to 7?

Reply:
Because in one of the failing tests with obs_id=9 had encounter with person_id=7